aiidalab_widgets_base/data/__init__.py

```diff
@@ -34,7 +34,10 @@ def __init__(self, value=0, description="Select functional group", **kwargs):
         self.style = {"description_width": "initial"}
         self.layout = {"width": "initial"}
         super().__init__(
-            value=value, description=description, options=FUNCTIONAL_GROUPS, **kwargs
+            value=value,
+            description=description,
+            options=[(key, value) for key, value in FUNCTIONAL_GROUPS.items()],
```

Comment:
```suggestion
            options=tuple([(key, value) for key, value in FUNCTIONAL_GROUPS.items()])
```

Reply:
I did not use this suggestion as is since in generates an unneeded list.